app/src/main/java/com/example/uiwidgets/UIWidgetsA.kt

```diff
@@ -26,7 +26,7 @@ class UIWidgetsA : AppCompatActivity() {
         val layouut = layoutInflater.inflate(R.layout.custom_toast, findViewById(R.id.custom_wrap))
 
         binding.back.setOnClickListener {
-            val intent = Intent(this, MainActivity::class.java)
+            val intent = Intent(this, Tabbed_app::class.java)
```

Comment:
Tabbeb_app? never use underscore for the class name, only use underscore in xmls